periodic/periodic.go

```diff
@@ -564,6 +564,9 @@ func getJitter(t time.Duration) time.Duration {
 		return time.Duration(0)
 	}
 	i := int64(t / 10)
```

Comment:
this probably should be something like
```golang
func getJitter(t time.Duration) time.Duration {)
   i:= int64(float64(t)/10.+ 0.5) // round to nearest instead of truncate
   if i <= 0 {
 	return time.Duration(0)
   }
   j := rand.Int63n(2*i) - i
   return time.Duration(j)
}
```
ie not test for 0 twice
(could also use math.Ceil and maybe use /5 and 2*i - i/2 instead of i/2 and -i) or use rand.Float64()...)

Reply:
or could check for <= 5 (or 10) at the beginning

Reply:
maybe something like:

```
func getJitter(t time.Duration) time.Duration {
   if t < 10 {
      return time.Duration(0)
   }
   i:= int64(t/10)    # if t < 10 check to make sure i > 0
   j := rand.Int63n(2*i) - i
   return time.Duration(j)
}
```

do not add `+0.5`, since when `float64(t)/10+ 0.5` is a constant, golang will throw `constant xx.xx truncated to integer`compile error,  as the constant does not fit into the specified type (int64).

Reply:
odd, /10. should keep everything float until after the int64() but not a huge deal

I wonder if you can you add a test that recreates the panic before the fix and shows the fix but it may be tricky to get sleep to be < 10 yet > 0

Reply:
https://play.golang.org/p/R_YEhoP_Qpr

works fine

Reply:
weird, i cannot reproduce the previous error i got. Yes, that looks fine. Will update!

Reply:
when I pass:

```
t := 6
i := int64(float64(t)/10 + 0.5)
```

or 

```
t := float64(6)
i := int64(t/10 + 0.5)
```

it works

But when I pass:
i := int64(float64(6)/10 + 0.5)

got ` constant 1.1 truncated to integer`